WordPress/src/test/java/org/wordpress/android/ui/jetpack/restore/RestoreDetailsViewModelTest.kt

```diff
@@ -0,0 +1,26 @@
+package org.wordpress.android.ui.jetpack.restore
```

Comment:
Warning (⚠️): This test resides in the wrong package. Consider moving that to an inner `details` package to match that with the `main` source set structure.

Reply:
I didn't put any of the tests into packages. I shall task this separately because it would need to be done for backup download as well. Thanks.